src/mmw/js/src/modeling/views.js

```diff
@@ -826,19 +830,39 @@ var ScenarioToolbarView = Marionette.CompositeView.extend({
     },
 
     initialize: function(options) {
-        this.mergeOptions(options, ['model_package']);
+        this.mergeOptions(options, ['model_package', 'projectModel']);
+        this.scenario = this.projectModel.get('scenarios')
```

Comment:
This view was already a little confusing, but is now more so.  We're not using the `ScenariosCollection`, `model_package` and `projectModel` consistently:
- we use both `App.currentProject` and `projectModel`
- the `model_package` is on the `projectModel`, so we could drop the `model_package`...this already sort of happens because we switch between using `options.model_package` and `App.currentProject.get('model_package')` ([`L814`](https://github.com/WikiWatershed/model-my-watershed/pull/2211/files#diff-416520c38e4c168e3f58d3f5089772c4R814))
- here we're using `this.projectModel.get('scenarios`) when we could be using `this.collection`

Reply:
Yeah, I was following the conventions in the rest of `modeling/views.js`, but I agree it wasn't a good place to start from.  Added 5258224 to clean stuff up.  Also rebased on the updated `develop`